nupic/frameworks/opf/clamodel.py

```diff
@@ -361,7 +361,8 @@ def run(self, inputRecord):
 
     self.__numRunCalls += 1
 
-    self.__logger.debug("CLAModel.run() inputRecord=%s", (inputRecord))
+    if self.__logger.isEnabledFor(logging.DEBUG):
```

Comment:
i would have to test, but im not sure of just the second line did not print "debug" always, while now only if set-up DEBUG ...?